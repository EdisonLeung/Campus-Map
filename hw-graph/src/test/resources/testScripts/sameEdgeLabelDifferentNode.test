# Tests adding 2 edges with same label to different nodes

# Create graph
CreateGraph sameEdge

# Add nodes
AddNode sameEdge a
AddNode sameEdge b

# adds 2 edges with same label but to different nodes
AddEdge sameEdge a b AB
AddEdge sameEdge b a AB

# Print nodes and children
ListNodes sameEdge
ListChildren sameEdge a
ListChildren sameEdge b