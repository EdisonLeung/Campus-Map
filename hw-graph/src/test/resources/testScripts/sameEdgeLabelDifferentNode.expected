# Tests adding 2 edges with same label to different nodes

# Create graph
created graph sameEdge

# Add nodes
added node a to sameEdge
added node b to sameEdge

# adds 2 edges with same label but to different nodes
added edge AB from a to b in sameEdge
added edge AB from b to a in sameEdge

# Print nodes and children
sameEdge contains: a b
the children of a in sameEdge are: b(AB)
the children of b in sameEdge are: a(AB)